lib/server/index.js

```diff
@@ -151,6 +151,9 @@ class Server extends EventEmitter {
     app.get('/testem.js', (req, res) => {
       this.serveTestemClientJs(req, res);
     });
+    app.get('/testem.cjs', (req, res) => {
```

Comment:
This shouldn't be needed as the `cjs` config file should only be read by the server and not be exposed to the client.

Reply:
ah nice -- I got confused by the file + path being named the same. I just verified this locally. Will remove.

Reply:
updated